pkgs/by-name/me/melonDS/package.nix

```diff
@@ -88,6 +100,6 @@ stdenv.mkDerivation (finalAttrs: {
       benley
       shamilton
     ];
-    platforms = lib.platforms.linux;
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

Comment:
Is there any Darwin maintainer to keep this running? 
I am tired of Darwin bitrot.